tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = py27
+
+
+[testenv]
+basepython = python2.7
+deps =
+    coverage
+    flake8
+    flake8_docstrings
+    mock
+    pytest
+    pytest-capturelog
+    Flask==0.10.1
+    Jinja2==2.7.3
+    MarkupSafe==0.23
+    MySQL-python==1.2.5
+    Werkzeug==0.9.6
+    argparse==1.2.1
+    itsdangerous==0.24
+    requests==2.9.1
+    wsgiref==0.1.2
```

Comment:
I just remove those lines :)

As to the flake8 session, them are error type we want o ignore(they are necessary I think).
e.g D103 require docstring in each public function,  D100 need docstring in each public module.
**One thing need to be mentioned is E731**. It's about we use lambda function in https://github.com/dminor/ouija/blob/master/src/updatedb.py#L130, which is E731 don't allow.
(you will see **src/updatedb.py:141:5: E731 do not assign a lambda expression, use a def if you remove it**) Do you want me fix that? I just thought we could forget it for now :)

@jmaher Thank you for you feedback! please review it again ;)